qiskit_optimization/translators/ising.py

```diff
@@ -42,6 +48,15 @@ def to_ising(quad_prog: QuadraticProgram) -> Tuple[OperatorBase, float]:
             in the problem.
         QiskitOptimizationError: If constraints exist in the problem.
     """
+    if opflow is None:
+        opflow = True
+        warn(
+            "`opflow` option of `to_ising` is not set explicitly. "
+            "It is set True now, but the default value will be changed to False. "
+            "We suggest setting `opflow=True` explicitly like `to_ising(qp, opflow=True)`.",
+            stacklevel=2,
```

Comment:
I'll comment here but it partly relates to the text in the release note. Here we default now to True so as to be backward compatible, but this default will be changed to False in a future (the next) release. The use of opflow operators will be deprecated at that point and will not be the default - presumably at that time, here we will detect anyone setting True explicitly and warn to change to False, or preferably remove their explicit setting since the opflow parameter will be removed after the deprecation period. We should recommend moving away from using opflow operators, which can be done right now by passing False here.